### ISSUE TITLE
Cannot download some youtube video

### ISSUE THREAD

**Issue by boygiandi** [streamlink/streamlink#3226]
## Bug Report

- [ x] This is a bug report and I have read the contribution guidelines.

### Description
I tried to download this video link "https://www.youtube.com/watch?v=KTkYNpZGuUo&feature=youtu.be" but failed.

```
[root@ns3161770 ~]# streamlink  "https://www.youtube.com/watch?v=KTkYNpZGuUo&feature=youtu.be" '480p'
/usr/lib/python2.7/site-packages/streamlink_cli/main.py:760: PythonDeprecatedWarning: Python 2.7 has reached the end of its life.  A future version of streamlink will drop support for Python 2.7. Please upgrade your Python to at least 3.5.
  streamlink = Streamlink({"user-input-requester": ConsoleUserInputRequester(console)})
[cli][info] streamlink is running as root! Be careful!
[cli][info] Found matching plugin youtube for URL https://www.youtube.com/watch?v=KTkYNpZGuUo&feature=youtu.be
[cli][info] Available streams: audio_mp4a, audio_opus, 144p (worst), 240p, 480p (best)
[cli][info] Opening stream: 480p (muxed-stream)
[cli][error] Try 1/1: Could not open stream <Stream()> (Could not open stream: Unable to open URL: https://r4---sn-25glen7y.googlevideo.com/videoplayback?expire=1601908916&ei=VNx6X4CYDZaJ7gOsrZHwDA&ip=51.89.65.74&id=o-AG1w7wv10KnZlmgdXWWeWoDKmSFuIzxY2RQcXW9qvTzd&itag=135&aitags=133%2C134%2C135%2C160%2C242%2C243%2C244%2C278&source=yt_otf&requiressl=yes&mh=HG&mm=31%2C26&mn=sn-25glen7y%2Csn-4g5ednss&ms=au%2Conr&mv=m&mvi=4&pl=23&initcwndbps=301250&vprv=1&mime=video%2Fmp4&otf=1&otfp=1&dur=0.000&lmt=1601766109378938&mt=1601887247&fvip=4&keepalive=yes&fexp=23915654&c=WEB&sparams=expire%2Cei%2Cip%2Cid%2Caitags%2Csource%2Crequiressl%2Cvprv%2Cmime%2Cotf%2Cotfp%2Cdur%2Clmt&sig=AOq0QJ8wRQIhAPmEUD1H5MGJpXN0OmXUPumrXSEBJcspNr1T-ekgbb5uAiBtcdwdc65kj6MmbOLr3uYvZP8Myz5R8C1VuMyWD2QYbA%3D%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRgIhALFvqm0MBXtvDsmuDK9MIU1d7YA7z6srqvs1la8midNrAiEA-X9HKzQGC8l4TvM6kHshD9h044LFHGZZGkBZZrrEdFE%3D (404 Client Error: Not Found for url: https://r4---sn-25glen7y.googlevideo.com/videoplayback?expire=1601908916&ei=VNx6X4CYDZaJ7gOsrZHwDA&ip=51.89.65.74&id=o-AG1w7wv10KnZlmgdXWWeWoDKmSFuIzxY2RQcXW9qvTzd&itag=135&aitags=133%2C134%2C135%2C160%2C242%2C243%2C244%2C278&source=yt_otf&requiressl=yes&mh=HG&mm=31%2C26&mn=sn-25glen7y%2Csn-4g5ednss&ms=au%2Conr&mv=m&mvi=4&pl=23&initcwndbps=301250&vprv=1&mime=video%2Fmp4&otf=1&otfp=1&dur=0.000&lmt=1601766109378938&mt=1601887247&fvip=4&keepalive=yes&fexp=23915654&c=WEB&sparams=expire%2Cei%2Cip%2Cid%2Caitags%2Csource%2Crequiressl%2Cvprv%2Cmime%2Cotf%2Cotfp%2Cdur%2Clmt&sig=AOq0QJ8wRQIhAPmEUD1H5MGJpXN0OmXUPumrXSEBJcspNr1T-ekgbb5uAiBtcdwdc65kj6MmbOLr3uYvZP8Myz5R8C1VuMyWD2QYbA%3D%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRgIhALFvqm0MBXtvDsmuDK9MIU1d7YA7z6srqvs1la8midNrAiEA-X9HKzQGC8l4TvM6kHshD9h044LFHGZZGkBZZrrEdFE%3D))
error: Could not open stream <Stream()>, tried 1 times, exiting
```

**Comment by johnthecracker** [comment]
Yes, you can but with youtube-dl.

**Comment by bastimeyer** [comment]
> root@ns3161770
> 404 Client Error: Not Found for url

This is either an issue on Youtube's servers in your region, or your server you're running Streamlink on got blocked by Youtube for abuse.
Duplicate of #2978

From a regular private user IP address, this is working fine:
```
$ streamlink -l debug 'https://www.youtube.com/watch?v=KTkYNpZGuUo'
[cli][debug] OS:         Linux-5.9.0-rc8-1-git-x86_64-with-glibc2.2.5
[cli][debug] Python:     3.8.5
[cli][debug] Streamlink: 1.6.0+23.g6de76fc
[cli][debug] Requests(2.24.0), Socks(1.7.1), Websocket(0.56.0)
[cli][info] Found matching plugin youtube for URL https://www.youtube.com/watch?v=KTkYNpZGuUo
[plugin.youtube][debug] Video ID from URL
[plugin.youtube][debug] Using video ID: KTkYNpZGuUo
[plugin.youtube][debug] get_video_info - 1: Found data
[plugin.youtube][debug] MuxedStream: v 135 a 251 = 480p
[plugin.youtube][debug] MuxedStream: v 133 a 251 = 240p
[plugin.youtube][debug] MuxedStream: v 160 a 251 = 144p
Available streams: audio_mp4a, audio_opus, 144p (worst), 240p, 480p (best)
```

**Comment by mkbloke** [comment]
There's maybe something weird about that video.

Did it actually play for you, @bastimeyer?  There's no problem getting the main playlist, but I see the same 404s from both the UK and Argentina when actually trying to play it with any of the given video qualities...

**Comment by bastimeyer** [comment]
Apologies, it does indeed not work.
However, it's a duplicate of #2909
